docs/tasks/configure-pod-container/share-process-namespace.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Share Process Namespace between Containers in a Pod
+min-kubernetes-server-version: v1.10
+approvers:
+- dawnchen
+- verb
+---
+
+{% capture overview %}
+
+{% include feature-state-alpha.md %}
+
+This page shows how to configure process namespace sharing for a pod. When
+process namespace sharing is enabled, processes in a container will be visible
+to all other containers in that pod.
+
+This can be useful for cooperating containers, such as a log handler sidecar
+container, or troubleshooting container images that don't include debugging
+utilities like a shell.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+A special **alpha** feature gate `PodShareProcessNamespace` has to be set to
+true across the system: `--feature-gates=PodShareProcessNamespace=true`.
```

Comment:
"A special **alpha** feature gate, `PodShareProcessNamespace`, must be set to true ..."

Reply:
Done